scripts/provision/provision.sh

```diff
@@ -0,0 +1,250 @@
+#!/bin/bash
+#
+# Copyright (c) 2018
+# Mainflux
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+###
+# Core Metadata
+###
+printf "> Provisioning Addressibles\n"
+
+# Addressable for the Device Service
```

Comment:
Why not use edgexfoundry/device-sdk-go/example/simple instead of creating a device service? Although is still in alpha state, this will simplify the provisioning script and add visibility to edgexfoundry/device-sdk-go